src/http_tracker_connection.cpp

```diff
@@ -333,7 +333,11 @@ namespace libtorrent
 
 		tracker_response resp = parse_tracker_response(data, size, ecode
 			, tracker_req().kind == tracker_request::scrape_request
-			, tracker_req().info_hash);
+			, tracker_req().info_hash
+#if TORRENT_USE_I2P
+			, tracker_req().i2pconn != NULL
```

Comment:
Do you have a suggestion for a namespace to put those flags into? *request* doesn't appeal to me as in `request::scrape | request::i2p`.